README.md

```diff
@@ -154,3 +154,9 @@ Our re-written version of `cs-console` does the following:
   through to gdb.
 
 * Forward any data sent by gdb through to the galileo board.
+
+## Known Issues
+
+1. **Don't use GDB Plugins with this project**
```

Comment:
nit: I'd just change this to "Caution if using GDB Plugins with...". 

Some plugins might work, others may not like in your case. It's behavior we never really tested.
Thanks for bringing it to our attention (:

Reply:
Good idea, changed this up so it's an actual suggestion :)